src/main/resources/application.properties

```diff
@@ -137,3 +137,11 @@ management.health.elasticsearch.enabled=false
 management.health.status.order=DOWN, OUT_OF_SERVICE, UNKNOWN, UP, OUT_OF_SYNC, PROBLEM, COULD_NOT_CONNECT
 
 spring.banner.location=classpath:ccd-data-store-banner.txt
+
+# OAuth2
+
+ccd.oauth2.client_id: ${OAUTH2_CLIENT_ID:ccd_gateway}
+ccd.oauth2.client_secret: ${OAUTH2_CLIENT_SECRET:OOOOOOOOOOOOOOOO}
+ccd.oauth2.redirect_url: ${OAUTH2_REDIRECT_URL:http://localhost:3451/oauth2redirect}
+ccd.cache.warm.up.email: ${CCD_CACHE_WARM_UP_EMAIL:ccdcachewarmer@gmail.com}
```

Comment:
Please no gmail addresses.  See https://tools.hmcts.net/confluence/display/RCCD/Automation+and+CCD

Reply:
Thanks. I will address it with @hemantt as he was the creator.

Reply:
New email were created following the guidelines.

Reply:
However, @hemantt is not sure how he can confirm registration via email as he won't have access to that email account.

Reply:
Fine with this being temporary as long as there is a ticket to resolve this.